src/app.js

```diff
@@ -30,4 +33,11 @@ vorpal
     callback();
   });
 
+vorpal
```

Comment:
Even if its self explanatory I think we should add comments anyway so we get good grades.

Reply:
Added more documentation.